### PR TITLE
Feature/add stg models

### PR DESCRIPTION
**Pull request by christineberger**
* Added sources for the staging models to reference in src_schema.yml
* Added staging models (prefixed with stg_)
* Added tests for staging models (stg_schema_yml)

### CODE REVIEW

File: models/staging/stg_customers.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+with source as (
+    select * from {{ source('dbt_christine', 'customers') }}
+)
+
```
Comment (aescay) [inline models/staging/stg_customers.sql:23]: best to rename this following the style guide naming conventions (as well as the rest of the staging tables)

**Review by aescay — CHANGES_REQUESTED** [review]
rename columns in staging to follow the prescribed style guide https://github.com/fishtown-analytics/corp/blob/master/dbt_coding_conventions.md

File: models/staging/stg_customers.sql (hunk 1 above)
Comment (christineberger) [reply models/staging/stg_customers.sql:23]: Renamed columns in stg models and tested:
Screen Shot 2020-04-05 at 12 19 26 AM

**Review by aescay — APPROVED** [review]
Great! 👍

**Comment by aescay** [comment]
Great catch on timestamps acting as just dates and converting it out 💯

**Pull request merged**